wrappers/map.js

```diff
@@ -71,8 +71,17 @@ class Map extends EventEmitter {
         }
       }
     }
-    if (config.icon)
-      options.pointToLayer = (feature, latLng) => L.marker(latLng, { icon: L.icon(config.icon(feature.properties)) })
+    if (config.icon) {
+      options.pointToLayer = (feature, latLng) => {
+
+        // apply marker style if defined, otherwise use leaflet default
+        if (typeof config.icon(feature.properties) !== 'undefined') {
```

Comment:
Wow, it took me a while to figure out where the `icon` function is defined and what return types it could have. We really shouldn't be naming that function in that way. Maybe `stylesForIcon` or something like that? At least it would look like a function instead of a property accessor. Anyway, not really a comment for this PR, more of an overall comment for our past selves @digitalsadhu.